SConstruct

```diff
@@ -1388,7 +1388,10 @@ def getPythonConfig( env, flags ) :
 	if f.returncode :
 		sys.stderr.write( "ERROR : Error running \"%s\".\n%s" % ( env["PYTHON_CONFIG"], stdErr ) )
 		Exit( 1 )
-	return r
+	if( sys.version_info >= (3, 0) ):
```

Comment:
The way I fixed this before was to pass `universal_newlines = True`, which is the obscure way of saying "give me a string back, no matter if i'm in python 2 or 3" : https://github.com/ImageEngine/cortex/commit/57d3ac0ffc8aaac552dd1ffd7cbfce58a70b3c92. I think the extra brevity provided by that approach is probably worth it,

Reply:
Cheers for that, worked great.